src/tls13.c

```diff
@@ -2835,19 +2848,55 @@ int SendTls13ClientHello(WOLFSSL* ssl)
         return SUITES_ERROR;
     }
 
+#ifdef WOLFSSL_ASYNC_CRYPT
+    ret = wolfSSL_AsyncPop(ssl, &ssl->options.asyncState);
+    if (ret != WC_NOT_PENDING_E) {
+        /* Check for error */
+        if (ret < 0)
+            return ret;
+    }
+    else
+#endif
+    {
+        /* Reset state */
+        ssl->options.asyncState = TLS_ASYNC_BEGIN;
+        XMEMSET(args, 0, sizeof(Sch13Args));
+    }
+
+    switch (ssl->options.asyncState) {
+    case TLS_ASYNC_BEGIN:
+    {
+    args->idx = RECORD_HEADER_SZ + HANDSHAKE_HEADER_SZ;
```

Comment:
4 spaces in.

Reply:
I did not add spacing to keep the line count low and so the diff is minimal.